packages/pegasus/src/types.js

```diff
@@ -111,3 +112,8 @@
  * @property {ConnectionHandler} handler
  * @property {Subscription<PegasusConnection>} subscription
  */
+
+/**
+ * @typedef {object} SenderOptions
+ * @property {string} sender the sender to attach to ics transfer packet
```

Comment:
I'd prefer making this `opt.sender` property optional, and encapsulating setting the default within `ics20.js`.

```suggestion
 * @property {string} [sender] the sender address attached to the packet to receive any refund
```